playwright-tests/tests/dashboard.treasury-devdao.near.spec.js

```diff
@@ -8,7 +8,18 @@ test.afterEach(async ({ page }, testInfo) => {
 
 test("should go to trustees dashboard", async ({ page, instanceAccount }) => {
   await page.goto(`/${instanceAccount}/widget/app?page=dashboard`);
+
+  await expect(
+    await page.locator("div.h3").filter({ hasText: "Dashboard" })
+  ).toBeVisible();
+
   await expect(
-    await page.getByRole("heading", { name: "Dashboard" })
+    await page.getByText("Treasury Assets: Sputnik DAO")
   ).toBeVisible();
+
+  await expect(await page.locator("iframe.chart")).toBeVisible();
+  await page.mouse.move(600, 500);
+  await page.mouse.move(1000, 500);
+
+  await page.locator("div[role='button']").filter({ hasText: "1H" }).click();
```

Comment:
Please insert an assertion that also the time labels changes when changing the time resolution on the charts